Task1BLINK.ino

```diff
@@ -0,0 +1,18 @@
+/*This sketch meets the Acceptance criteria of this task: https://trello.com/c/ba783WmU/2-blink-an-led
+*/
+ 
+void setup() {
+
+  pinMode (13, OUTPUT);
+}
+
+
+void loop() { /* this function runs untill you press off*/ 
+
+  digitalWrite(13, HIGH);/*light on*/ 
+  delay(1000); /*1 second delay*/
+  digitalWrite(13, LOW);/*light off*/ 
+  delay(2000); /* 2 second delay*/ 
+}
+
+
```

Comment:
you should set the pin number as a variable, so you dont have to write out 13 every time you need the pin number, this will also make it easier to change if you change board where the pin is different